agent/lib/kontena/websocket_client.rb

```diff
@@ -101,26 +100,27 @@ def send_message(msg)
       error "failed to send message: #{exc.message}"
     end
 
+    def send_notification(method, params)
+      data = MessagePack.dump([2, method, params]).bytes
+      send_message(data)
+    end
+
     # @param [Faye::WebSocket::API::Event] event
     def on_open(event)
       ping_timer.cancel if ping_timer
       info 'connection established'
       @connected = true
       @connecting = false
+      Celluloid::Notifications.publish('websocket:connect', event)
```

Comment:
Good point.. have to look around and check where this is actually used. I think proper event here would be `websocket:open` or similar.